frontend/src/components/admin/testNotificationConfigMenu/TestNotificationConfigEdit.js

```diff
@@ -551,7 +551,7 @@ function TestNotificationConfigEdit() {
           <br />
           <div>
             <Grid fullWidth={true}>
-              <Column lg={14} md={6} sm={2}>
+              <Column lg={16} md={8} sm={4}>
```

Comment:
Why change the  column size for lg ?? , it will distort the layout on a large screen